torch/testing/_internal/common_nn.py

```diff
@@ -118,6 +118,16 @@ def get_weight(m):
         with_tf32=True,
         tf32_precision=0.005,
     ),
+    dict(
```

Comment:
FYI: "Eventually" these dicts will be removed in favor of `ModuleInfo` entries in `torch/testing/_internal/common_modules.py`. The entries there are used to auto-generate tests in `test/test_modules.py` that hit the same coverage as the tests using these dicts.

Reply:
ic... thanks for sharing this.